pkg/k8smgmt/cluster.go

```diff
@@ -394,6 +413,72 @@ func convertNodeResource(res v1.ResourceName, quantity resource.Quantity) (strin
 	return name, dec, nil
 }
 
+func getNodeGPUInfo(labels map[string]string) ([]*edgeproto.GPUResource, *edgeproto.GPUSoftwareInfo, error) {
+	// Note: we only support one GPU per node, as not sure
+	// what the labels would look like for multiple GPUs
+	// or if the gpu-operator even supports it.
```

Comment:
`nvidia.com/gpu.count` would be > 1 for that node output. But this should work transparently. I think it's safe to assume multiple gpus that are the same.

Reply:
Sorry, let me fix that comment, what I meant there was multiple GPU types, i.e. a A100 and a T4 on the same node. I believe the operator doesn't support that either currently.